### PR TITLE
Add Helm chart for ecr-viewer

### PR DESCRIPTION
**Pull request by emmastephenson**
# PULL REQUEST

## Summary
Creates a Helm chart for the ecr viewer.

[//]: # (REQUIRED)
## Related Issue
Fixes [#1084](https://github.com/CDCgov/phdi/issues/1084)

## Additional Information
Will test this locally. It shouldn't be merged until the ecr-viewer has a container image released on `phdi`.

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by emmastephenson** [comment]
Tested locally, and this chart should be working. Some things to note for local development in the future:
* To access the IP the service is running on, it's necessary to run `kubectl proxy` as well as `kubectl port-forward service/$SERVICE-NAME 3000:3000`. Then, you should be able to access the service at `127.0.0.1:3000`. 
* Unlike our other services, the eCR viewer is [exposed on port 3000](https://github.com/CDCgov/phdi/blob/main/containers/ecr-viewer/Dockerfile#L6), so the default port for the service also needs to be 3000.
* I couldn't get the database connection working locally, but it wasn't an issue with the Helm chart - it was just local networking between my Kind cluster and the Docker container that was running Postgres. (My test db connection string included `localhost:5432`, and the K8s cluster has no idea what "localhost" is.) I stopped trying to debug this because the database connection string _is_ included in the `env` vars on the service, and the networking considerations will be very different in a cloud environment - I doubt we'll be running Helm locally most of the time.

File: charts/ecr-viewer/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: ecr-viewer
+description: A Helm chart for the DIBBs eCR Viewer
+type: application
+version: 0.1.0
+appVersion: "1.1.10"
```
Comment (KennethSkylight) [inline charts/ecr-viewer/Chart.yaml:6]: Not sure if it really matters but the app version doesn't match what's in ingestion

Comment (KennethSkylight) [inline charts/ecr-viewer/Chart.yaml:6]: Not sure if it really matters but the app version doesn't match what's in ingestion

**Review by KennethSkylight — COMMENTED** [review]
LGTM just one app version change

File: charts/ecr-viewer/Chart.yaml (hunk 1 above)
Comment (emmastephenson) [reply charts/ecr-viewer/Chart.yaml]: Oops! It should be one version up to match what's currently published in `phdi`; fixed now.

**Review by marycrawford — COMMENTED** [review]
Do we need to add the `ecr-viewer` chart to the eks module as well?  In the [variable.tf](https://github.com/CDCgov/phdi-playground/blob/main/terraform/aws/implementation/modules/eks/variables.tf#L26-L35) file?  If we also updating Azure we would need to update this in the [main.tf](https://github.com/CDCgov/phdi-playground/blob/main/terraform/implementation/main.tf#L24-L34) file.

**Review by KennethSkylight — APPROVED** [review]

**Comment by emmastephenson** [comment]
> Do we need to add the `ecr-viewer` chart to the AWS eks module's [variable.tf](https://github.com/CDCgov/phdi-playground/blob/main/terraform/aws/implementation/modules/eks/variables.tf#L26-L35) file? Are we also updating Azure? If so, we may need to update this in the [main.tf](https://github.com/CDCgov/phdi-playground/blob/main/terraform/implementation/main.tf#L24-L34) file.

@marycrawford yes, we would want to add this ecr viewer chart to the AWS EKS instance! It'd be nice to add to Azure too, though that's less important in the short term.

**Pull request merged**